### PR TITLE
2.1.3 release

### PR DESCRIPTION
**Pull request by Rahmon**
### Checklist:

- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jeffpaul** [comment]
will want to bump https://github.com/10up/brightcove-video-connect/blob/7312968e658fb01decd05d325f8a56fa4302def2/package.json#L6 as well

**Comment by jeffpaul** [comment]
also https://github.com/10up/brightcove-video-connect/blob/7312968e658fb01decd05d325f8a56fa4302def2/brightcove-video-connect.php#L6

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [2.1.3] TBD
+### Fixed
+- Fix: Playlist player is not available
+
 ## [2.1.2] TBD
```
Comment (oscarssanchez) [inline CHANGELOG.md:11]: Let's add the release date here @Rahmon you can grab this info from: https://github.com/10up/brightcove-video-connect/releases/tag/2.1.2 

I usually leave the TBD to the most recent one, as sometimes the release date is not the same the releas PR is submitted

File: readme.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ Donate link: https://supporters.eff.org/donate
 Tags: brightcove, 10up, videos, video
 Requires at least: 4.2
 Tested up to: 5.6
-Stable tag: 2.1.2
```
Comment (oscarssanchez) [inline readme.txt:8]: Let's also add the changes to the Changelog like this @Rahmon : https://github.com/10up/brightcove-video-connect/pull/196/files#diff-9e6e4772050998a5c0dc3c61acf3dab0a7e594566171fa5746d6b62f9598efb6R25

**Review by oscarssanchez — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 2 above)
Comment (Rahmon) [reply readme.txt:8]: @oscarssanchez Updated.

File: CHANGELOG.md (hunk 1 above)
Comment (Rahmon) [reply CHANGELOG.md]: Updated @oscarssanchez

**Review by oscarssanchez — APPROVED** [review]

**Pull request merged**